x-pack/plugins/canvas/public/state/selectors/__tests__/workpad.js

```diff
@@ -198,6 +245,15 @@ describe('workpad selectors', () => {
     });
   });
 
+  describe('getGlobalFilterExpression', () => {
+    it('gets filters from all elements', () => {
+      const filters = selector.getGlobalFilterExpression(state);
+      expect(filters).to.equal(
+        'exactly value="beats" column="project" | timefilter column=@timestamp from=now-24h to=now'
```

Comment:
if two elements have the same filters, then that filter string will be duped in the globalFilterExpression.  Is there any value in deduping the array before doing the `.join(|)`?

Reply:
Possibly, but there's also no harm in just including it twice, just a small increase in expression execution time.

Reply:
@crob611 I should add, if you think it's worth doing, open an issue about it. It wouldn't be hard to add, but since this PR isn't a functional change, I'm not going to do it here.

Reply:
Yeah, I imagine the cost of deduping every time would outweigh any gains it would get us in the scenarios where filters are duplicated.